deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,62 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+.outer-col-1{
+    display: flex;
+    flex-direction: row;
+}
+
+.outer-col-2{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100vw;
+}
+
+.inner-row{
+    display: flex;
+    flex-direction: row;
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+.inner-col{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100vw;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+nav{
+    flex-basis: 15vw;
```

Comment:
Het aanspreken van de NAV was overbodig. Als je deze flex-basis property onder regel 11 in de class outer-col-1 van de nav had geplaatst was de uitkomst hetzelfde en blijft het netjes bij elkaar.